builders/get-started/networks/moonbeam-dev.md

```diff
@@ -67,6 +84,30 @@ Using Docker enables you to spin up a node in a matter of seconds. Once you have
 
 If successful, you should see an output showing an idle state waiting for blocks to be authored:
 
+<!-- termynal -->
+```bash
+2023-12-13 22:46:22 💻 Operating system: linux
+2023-12-13 22:46:22 💻 CPU architecture: x86_64
+2023-12-13 22:46:22 💻 Target environment: gnu
+2023-12-13 22:46:22 💻 CPU: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
+2023-12-13 22:46:22 💻 CPU cores: 12
+2023-12-13 22:46:22 💻 Memory: 7946MB
+2023-12-13 22:46:22 💻 Kernel: 6.4.16-linuxkit
+2023-12-13 22:46:22 💻 Linux distribution: Debian GNU/Linux 12 (bookworm)
+2023-12-13 22:46:22 💻 Virtual machine: yes
+2023-12-13 22:46:22 📦 Highest known block at #0
+2023-12-13 22:46:22 〽️ Prometheus exporter started at 127.0.0.1:9615
+2023-12-13 22:46:22 Running JSON-RPC server: addr=0.0.0.0:9944, allowed origins=["*"]
+2023-12-13 22:46:22 🏁 CPU score: 1000.18 MiBs
+2023-12-13 22:46:22 🏁 Memory score: 10.68 GiBs
+2023-12-13 22:46:22 🏁 Disk score (seq. writes): 1.55 GiBs
+2023-12-13 22:46:22 🏁 Disk score (rand. writes): 469.38 MiBs
+2023-12-13 22:46:22 Development Service Ready
+2023-12-13 22:46:27 💤 Idle (0 peers), best: #0 (0x6bfb…59a6), finalized #0 (0x6bfb…59a6), ⬇ 0 ⬆ 0
+2023-12-13 22:46:32 💤 Idle (0 peers), best: #0 (0x6bfb…59a6), finalized #0 (0x6bfb…59a6), ⬇ 0 ⬆ 0
+2023-12-13 22:46:37 💤 Idle (0 peers), best: #0 (0x6bfb…59a6), finalized #0 (0x6bfb…59a6), ⬇ 0 ⬆ 0
+```
+
 ![Docker - output shows blocks being produced](/images/builders/get-started/networks/moonbeam-dev/moonbeam-dev-2.png)
```

Comment:
Same thought here. The text looks nicer than the screenshot IMHO

Reply:
oops haha I was messing around with the mkdocs termnyal plugin and accidentally committed my local changes 😬